clang-tools-extra/docs/clang-tidy/checks/readability/use-std-min-max.rst

```diff
@@ -0,0 +1,32 @@
+.. title:: clang-tidy - readability-use-std-min-max
+
+readability-use-std-min-max
+===========================
+
+Replaces certain conditionals with ``std::min`` or ``std::max`` for readability,
+promoting use of standard library functions. Note: This may impact
```

Comment:
Please make sure that the first sentence is the same as in the header. The release notes (minus the second half of the sentence) and the header have the same one, so you would probably want to update this one with the one from the header.